Adding required perm (#2397)
MicrosoftDocs/microsoft-365-docs: microsoft-365/admin/manage/customize-the-app-launcher.md

  
## Add a custom tile to the app launcher

> 1. Log into the admin center as a Global Administrator, go to the **Settings** > **Org Settings** and choose **Organization profile** tab.

Reviewer: ```suggestion
1. Sign in to the admin center as a Global Administrator, go to **Settings** > **Org Settings**, and choose the **Organization profile** tab.
```
